File: TvShowServer/Model/network.py
class Network(object):

    def __init__(self,id, network):
        self.id = id
        self.network = network

    def getId(self):
        return self.id

    def getNetwork(self):
        return self.network

    def serialize(self):
        return {'id': self.id, 'network': self.network}

File: TvShowApp/app/src/main/java/com/gm/gislain/tvshowapp/Model/Network.java
package com.gm.gislain.tvshowapp.Model;

/**
 * Created by Gislain on 25/11/2016.
 */

public class Network {

    private int id;
    private String network;

    public static final String KEY_ID = "id";
    public static final String KEY_NETWORK = "network";

    public Network() {}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNetwork() {
        return network;
    }

    public void setNetwork(String network) {
        this.network = network;
    }
}
